Documentation/Contributing.md

```diff
@@ -0,0 +1,160 @@
+# Swift Package Manager: Quick Start
+Swift Package Manager (SwiftPM) is a tool for building, testing and managing Swift project dependencies.  
+In order to use it you will need Swift 3.0 or greater.  
+SwiftPM is also included in Xcode 8.0 and above.  
+For usage getting started: [https://swift.org/getting-started/#using-the-package-manager](https://swift.org/getting-started/#using-the-package-manager)  
+For overview and examples: [https://swift.org/package-manager](https://swift.org/package-manager/)
+
+## Code Contributions
+Everyone is welcome to contribute to SwiftPM, submitting fixes, enhancement etc.  
```

Comment:
We should add some text here about the different types of changes folks can make, something along the lines of

```markdown
There are several types of contributions one can make. Bug fixes, documentation and enhancements that do not materially change the user facing semantics of Swift Package Manager should be submitted directly as PR. 

Larger changes that do materially change the semantics of Swift Package Manager (e.g. changes to the manifest format or behavior) are required to go through [Swift Evolution Process](https://github.com/apple/swift-evolution/blob/master/process.md).
To see how previous evolution decisions for SwiftPM have been made check out https://github.com/apple/swift-package-manager/blob/main/Documentation/Internals/PackageManagerCommunityProposal.md

For more information about making contributions to the Swift project in general see [Swift Contribution Guide](https://swift.org/contributing/) 
```